conjur_api/http/api.py

```diff
@@ -504,6 +504,26 @@ async def rotate_personal_api_key(
                                          ssl_verification_metadata=self.ssl_verification_data)
         return response.text
 
+    async def set_authenticator_state(self, authenticator_id: str, enabled: bool) -> str:
+        """
+        This method sets the state of a given authenticator
+        """
+
+        params = {
+            'authenticator_id': authenticator_id
+        }
+        params.update(self._default_params)
+
+        body = f'enabled={"true" if enabled else "false"}'
```

Comment:
Maybe we can use `str(enabled).lower()` to get the same result. 
Not sure what is better, just throwing in idea

Reply:
Yes, I like it. Fixed, thanks!